clients/bazel-module-registry/src/main/kotlin/Model.kt

```diff
@@ -21,6 +21,8 @@ package org.ossreviewtoolkit.clients.bazelmoduleregistry
 
 import io.ks3.java.typealiases.UriAsString
 
+import java.net.URI
```

Comment:
Commit message:
- We usually write "source code" as two word.
- "won't" -> "will not"; but actually, the sentence leaves a bit unclear whether it describes the state before or after the change. Maybe just say "ORT should not query..."?
- "mod graph" -> `mod graph` (use backticks instead of quotes when referring to commands).